code/game/gamemodes/gang/dominator.dm

```diff
@@ -53,7 +53,7 @@
 			if(!warned && (time_remaining < 180))
 				warned = 1
 				var/area/domloc = get_area(loc)
-				gang.message_gangtools("Less than 3 minutes remain in hostile takeover. Defend your dominator at [domloc.map_name]!")
+				gang.message_gangtools("Less than 3 minutes remains in hostile takeover. Defend your dominator at [domloc.map_name]!")
```

Comment:
I've said my part and don't wish to continue arguing grammar rules over the correctness of a single letter based on how you dissect and augment a sentence so lets go with Joan, here are some third options off the top of my head:
`There are less than 3 minutes remaining until hostile takeover is complete.`
`Hostile takeover will be complete in less than 3 minutes.`
`Hostile takeover: T-minus 3 minutes and counting.` (Do this one cos space ( ͡° ͜ʖ ͡°))

And for a fourth option; is this message really needed? There's already a stat panel timer and no other system send a time dependent message when victory is close. Seems kinda unnecessary.